redaxo/src/core/lib/response.php

```diff
@@ -14,7 +14,8 @@ class rex_response
     const HTTP_FORBIDDEN = '403 Forbidden';
     const HTTP_UNAUTHORIZED = '401 Unauthorized';
     const HTTP_INTERNAL_ERROR = '500 Internal Server Error';
-
+    const HTTP_TEMPORARILY_UNAVAILABLE = '503 Service Temporarily Unavailable';
```

Comment:
Woher kommt der statustext für den 503?

Im netz finde ich zu 503 eher „Service Unavailable“
https://de.wikipedia.org/wiki/HTTP-Statuscode

Reply:
Manchmal wird für den 503 ein anderer Statustext verwendet.
Der nach [RFC 7231](https://tools.ietf.org/html/rfc7231#section-6.6.4) standardisierte Text ist "Service Unavailable"

Reply:
stimmt, meine Variante wird erst in Verbindung mit Retry-After: interessant